views/components/icons.njk

```diff
@@ -136,7 +136,7 @@
          viewBox="0 0 459 459" width="14" height="14"
          aria-labelledby="aria-i-bin-title"
     >
-        <title id="aria-i-bin-title">A waste bin</title>
+        <title id="aria-i-bin-title">A waste bin icon</title>
```

Comment:
Mind converting this icon to use the `icon` macro (see `iconSearch`) it will fix a11y issues complaining about the icon needing a unique id.